include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -656,6 +656,7 @@ struct SPhysicalDeviceLimits
     // we don't compare certain capabilities because they don't mean better/worse
     inline bool isSubsetOf(const SPhysicalDeviceLimits& _rhs) const
     {
+        if (minMaxImageDimension2D > _rhs.minMaxImageDimension2D) return false;
```

Comment:
you do realize this is a `constexpr` so its always the same, you need to omit those!